### PR TITLE
demonstrate #680

### PR DESCRIPTION
**Pull request by lneuhaus**
See #680

### CODE REVIEW

File: lmfit/models.py [hunk 1]
```diff
@@ -352,7 +352,9 @@ def __init__(self, independent_vars=['x'], prefix='', nan_policy='raise',
     def _set_paramhints_prefix(self):
         self.set_param_hint('amplitude', min=0)
         self.set_param_hint('frequency', min=0)
-        self.set_param_hint('shift', min=0, max=tau)
+        bugfix_active = False
+        if not bugfix_active:
+            self.set_param_hint('shift', min=0, max=tau)
```
Comment (lneuhaus) [inline lmfit/models.py:357]:
```suggestion
```
I put this here to demonstrate the bug Setting bugfix_active to True this will get tests passing.

**Comment by lneuhaus** [comment]
@Tillsten @newville did you mean to close this with #680 ?

**Comment by newville** [comment]
@lneuhaus  Were you expecting this code to be merged into the main branch?   That is, literally, what a PR is.    I would hope that is obvious that we would not merge this as it is.  

I sort of asked you multiple times to clarify #680 and now you've made a PR of code that is obviously not going to be merged into supported code.  If you want to improve the guessing for SineModel, that would be OK.  But if that was actually the case, then why didn't you say you wanted to do that and then do that in a PR?   Your goals might be genuinely trying to fix the code, but your responses to direct requests for clarification and your actions are actually consistent with *trying* to waste our time and being a troll.   Honestly, I'm just baffled.

Sorry, I don't really have time to entertain trolls or give tutorials on basic coding/open-source skills.   I'm spending way too much of my time trying to respond to you in a polite and respectful manner.  Please, either learn to respect us and our processes back, or go away.

**Comment by lneuhaus** [comment]
Sorry. Will continue in my own fork. Im not a troll. Sorry again. Sorry sorry sorry.

**Pull request closed without merging**